crates/reporting/tests/test_reporting.rs

```diff
@@ -2528,11 +2534,11 @@ mod test_reporting {
 
     This `a` value is a:
 
-        [A]
+        […]
 
     But the type annotation on `f` says it should be:
 
-        [A, B]
+        [B, …]
```

Comment:
This can come up in more scenarios than that though...like for example if you have two tag unions with several tags, and they're exactly the same except one of them has a tag the other is missing. So I think if `[…]` is confusing, it would still come up.

Given that, I'm inclined to try it out and see if people find it confusing in practice. If so, we can investigate potential solutions then. How does that sound?

Reply:
Sounds reasonable to me. I was a bit confused reading `[...]` and thought "oh, an open union?", but that might just be me. LTGM 🚢